templates/partners/find-a-partner.html

```diff
@@ -21,22 +21,22 @@ <h1>Find a Canonical partner</h1>
       <div class="col-3">
         <form class="p-search-box u-no-margin--bottom">
           <input class="p-search-box__input js-find-a-partner__search-input" name="search" id="search" placeholder="Search" required="" type="search">
-          <button type="reset" class="p-search-box__reset u-no-margin--right" alt="reset"><i class="p-icon--close"></i></button>
+          <button type="reset" class="p-search-box__reset u-no-margin--right" alt="reset"><i class="p-notification__close"></i></button>
```

Comment:
Why was this changed? This isn't a notification, and the icon doesn't show when it should.
```suggestion
          <button type="reset" class="p-search-box__reset u-no-margin--right" alt="reset"><i class="p-icon--close"></i></button>
```